java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -777,6 +778,31 @@ protected Map<String, Object> handleLcsIdxResponse(Map<String, Object> response)
         return response;
     }
 
+    /**
+     * Reset the current connection password with a new password.
+     *
+     * <p>This method is useful in scenarios where the server password has changed or when utilizing
+     * short-lived passwords for enhanced security. It allows the client to update its password to
+     * reconnect upon disconnection without the need to recreate the client instance. This ensures
+     * that the internal reconnection mechanism can handle reconnection seamlessly, preventing the
+     * loss of in-flight commands.
+     *
+     * @apiNote This method updates the client's internal password configuration and does not perform
+     *     password rotation on the server side.
+     * @param password A new password to set or empty value to remove the password.
+     * @param mode Password update mode, see {@link PasswordUpdateMode}.
```

Comment:
See my comment below.
You asked about it in my pr and I answered that I don't understand, reading the impl I understand your question.
The API can't be entirely different functionality between the clients, even if your understanding were right.
We didn't even have a discussion about it, it ended by a question mark hanged, so its even not a disagreement.

More important, deciding to abstract something differently in one of the bindings, while it is not VK command but an internal feature doesnt make any sense, since the logic is not in the binding, its in the core. Abstracting it "like" it does something is completly redundent. Your abstraction dont affect the behaviour.

I insist that in any case that theres a feature added, and implementing the binding is seperated by the team, we will not decide to implement something differently ever. If the disagreement is so strong you can drop the implementation, but never different.

In the case of auth, theres work on many feature, while thinking on implemantetion in the binding i needed to make sure im not introducing something that will create a need for breaking changes later when logic is added.
If you implement diffrently and i missed it, you might not know why i choose to do in some way, and in the next phase all the bindings will need to have workarounds to not break your decision or being seperate completly.